dbterd/adapters/sources/file_io.py

```diff
@@ -0,0 +1,23 @@
+import json
```

Comment:
Would you mind to merge this into `helpers/file.py` or move into `helpers` directory? Thanks